### DISCUSSION TITLE
Docker images do no longer work with docker version 25

### DISCUSSION THREAD

**Discussion by renma** [eclipse-cdt/cdt#709] [Q&A]
My eclipse version is 2023-12 (4.30.0).

I updated to docker version 25 today and compiling with docker images no longer work. When opening the docker explorer the following error message is displayed...
```
An internal error occurred during:
"Opening connection 'unix:///var/run/docker.sock'[unix:///var/run/docker.sock]...".
Cannot invoke "String.startsWith(String)" because "s" is null
```
Docker containers are no longer displayed at all...

Can anyone confirm this and maybe help what could be done.
I have downgraded to docker 24.0.7 for now.

Thank you very much for your attention.

**Comment by akurtakov** [comment]
This is issue https://github.com/eclipse-linuxtools/org.eclipse.linuxtools/issues/309 . Any help is more than appreciated as it already costs me a lot of time and honestly I rely on podman(not supported by the plugin) locally so community jumping in is more than needed.
